pkg/utils/utils.go

```diff
@@ -50,3 +52,14 @@ func SaveToFile(fileName string, content []byte) (string, error) {
 	glog.Infof("Saved App Gateway config to %s", tempFile.Name())
 	return tempFile.Name(), nil
 }
+
+// GetHashCode generates hashcode of given type
+func GetHashCode(i interface{}) string {
```

Comment:
We can and should compress this a bit.
I'd also recommend not ignoring the Marshall error.

I'd do something like this:
```
func GetHashCode(s string) (string, error) {
	jsonBytes, err := json.Marshal(s)
	if err != nil {
		glog.Errorf("Failed MD5 hashing %s", s)
		return "", err
	}
	return fmt.Sprintf("%x", md5.Sum(jsonBytes)), nil
}
```

Reply:
I have updated this.